tests/integration-tests/tests/common/utils.py

```diff
@@ -401,3 +401,48 @@ def wait_process_completion(remote_command_executor, pid):
         raise Exception("The process is still running")
     else:
         return result.stdout.strip()
+
+
+def get_deployed_config_version(cluster, compute_node_ip: str = None):
+    """Retrieves the cluster config version deployed on the cluster node from its dna.json
```

Comment:
Suggestion: we could try to follow [multi-line doc-strings format](https://peps.python.org/pep-0257/#multi-line-docstrings).

In integ tests this is not enforced by our linters, but mentioning it because in the code it's forced.